src/pages/my.tsx

```diff
@@ -15,6 +17,8 @@ interface MyPageProps {
 
 export default function MyPage({ user } : MyPageProps) : JSX.Element {
     return (
-        <h1>Hello, { user.first_name }</h1>
+        <Heading level={ 1 }>
+            <Msg id="pages.my.welcome"/>, { user.first_name }
```

Comment:
Pleaes pass the user's name as a value to `<Msg/>` instead of appending it like this. Different languages/translators might want to position the name differently.